factory/factory_constants.tpt2

```diff
@@ -72,6 +72,16 @@
 
 {category(crft, chip, block, void_sapling)}
 
+; ----- EVENT ITEMS -----
+
+{make(anti_pumpkin, 1, "pumpkin.anti")}
```

Comment:
By using `make` for all of these, they will all attempt to be crafted - even things like anti_pumpkins which can't be crafted, only produced via machine. I suppose technically that doesn't hurt anything (I was slightly surprised that this worked at first, but thinking about it trying to craft an uncraftable item is a no-op), but it will make the script a bit bigger and slower, which is less than ideal.

However, as noted at the top all the `make` items have to come together in a single block, which means doing this properly involves splitting the items up. I think that's actually good though - IMO it's better to spread the pumpkin items into their natural groups, instead of creating a new event group for them. I.e. I expect people will look for pumpkin_producers under PROD instead of EVNT.

Other than that, this seems reasonable, and I think I'd be willing to take the hit of the extra script for the time being.

Reply:
Admittedly, I haven't been playing TPT2 that long, so I don't know if the recipes remain available outside of the event or not.  As such, I thought that it might be helpful if the event-related recipes were grouped separately, to potentially reduce some confusion in that regard.  At the same time, I can easily see someone looking for pumpkin_producers with the other producers.  Ultimately, it is your package, so I defer to your judgement.

In terms of moving the items to the appropriate existing groups, do you have a preference on the ordering?  For instance, I know that the factory floor two producers come after the other producers, rather than being strictly alphabetical.

As to the usage of `make`, I'm a touch annoyed with myself, because I THOUGHT I had looked for similar items (e.g. the entries for the regular plates), but either I hadn't, or I did and promptly forgot.

Reply:
I don't have much of a preference. I'm probably going to reorder the producers anyway, since I'm going to change them from colors to building names.

Reply:
Adjusted as requested